beacon-chain/blockchain/head_test.go

```diff
@@ -18,15 +20,19 @@ func TestSaveHead_Same(t *testing.T) {
 	defer testDB.TeardownDB(t, db)
 	service := setupBeaconChain(t, db)
 
-	service.headSlot = 0
+	identifier := [40]byte{}
 	r := [32]byte{'A'}
-	service.canonicalRoots[0] = r[:]
+
+	copy(identifier[:8], bytesutil.Bytes8(0))
```

Comment:
I think a lot of these can be refactored into its own function to reduce disparity down the line